.codecov.yml

```diff
@@ -5,3 +5,8 @@ codecov:
   # DAutoTester and auto-tester are not in the default list
   ci:
     - !travis
+
+coverage:
+  fixes:
+    - "test/.*/::src/"
+    - "fail_compilation/::src/"
```

Comment:
```
src/file.d
test/blah/file.d

# path=file.lst
.....
```

The issue is that codecov will resolve the path `file.lst` to `test/blah/file.d`.